lib/matplotlib/path.py

```diff
@@ -552,6 +571,17 @@ def contains_points(self, points, transform=None, radius=0.0):
         Returns
         -------
         length-N bool array
+
+        Notes
+        -----
+        The current algorithm has some limitations:
+
+        - The result is undefined for points exactly at the boundary
+          (including *radius*).
+        - The result is undefined if there is no enclosed area, i.e. all
+          vertices are on a straight line.
+        - If bounding lines start to cross each other due to *radius* shift,
+          the result is not guaranteed to be correct.
```

Comment:
Should this section just read, "this function has various limitations, for details see `contains_point`"? I know we bend over backward to repeat keyword args and so forth in every docstring for plotting functions, but this is more of a "power-user"-facing function than a user-facing/plotting function, so I'm personally comfortable allowing the redirection (and we have historically used such redirection elsewhere in the codebase, I think?)

Reply:
I think either way is fine here.  Just because power-users _can_ chase references, does not mean we should make them.

Reply:
No strong opinion. It's unlikely to change (little danger of becoming invalid) and explicit is slightly better than an indirection. So for now (personal preference), I'd leave this explicit.